onnxruntime/python/tools/transformers/benchmark_gpt2.py

```diff
@@ -439,14 +440,13 @@ def export_onnx(model, config, tokenizer, device, output_dir, use_LMHead, use_at
     return export_model_path
 
 
-def create_onnxruntime_session(onnx_model_path, use_gpu, verbose):
+def create_onnxruntime_session(onnx_model_path, use_gpu, verbose, thread_num):
     session = None
     try:
         from onnxruntime import SessionOptions, InferenceSession
         sess_options = SessionOptions()
-        if not use_gpu:
-            sess_options.intra_op_num_threads = psutil.cpu_count(logical=True)
-            logger.debug(f"Session option: intra_op_num_threads={sess_options.intra_op_num_threads}")
+        sess_options.intra_op_num_threads = thread_num
```

Comment:
intra_op_num_threads only works for onnxruntime-gpu but not onnxruntime page. Update OMP_NUM_THREADS environ variable for cpu package (before importing onnxruntime.

Reply:
same as the previous one.